app/views/datasets/show.html.erb

```diff
@@ -19,7 +19,7 @@
       <%= render "supporting_docs" %>
   <% end %>
 
-  <% if @dataset['organisation']['contact_name'] || @dataset['organisation']['contact_email'] %>
+  <% if @dataset['organisation']['contact_email'].present? %>
```

Comment:
This seems sensible, but maybe need to run passed gurinder/blanca? I think there is a general vibe that is if there's info display it - perhaps it's useful to know the name of the contact person?